connector_search_engine/models/se_backend_spec_abstract.py

```diff
@@ -12,8 +12,7 @@ class SeBackendSpecAbstract(models.AbstractModel):
 
     name = fields.Char(
         related='se_backend_id.name',
-        store=True,
-        required=False,
+        # store=True,
```

Comment:
If put store=True I get:

```
ParseError: "null value in column "name" violates not-null constraint
DETAIL:  Failing row contains (1, 1, null, 1, 1, null, 2018-12-10 18:39:57.431613, 2018-12-10 18:39:57.431613).
" while parsing /odoo/external-src/connector-search-engine/connector_algolia/demo/backend_demo.xml:9, near
<record id="backend_1" model="se.backend.algolia">
        <field name="se_backend_id" ref="se_backend_1"/>
    </record>
```

whatever se.backend name required or not ?

No more commit of me in this PR : impossible to me to improve